Updating patch build process and adding to release deployment.

jmdegeorge opened the pull request:
Completed patch build updates and revisions to bat update script to utilize new patch location and clear dropins/lib folder when updating.

jmdegeorge committed:
Updating patch build process and adding to release deployment.
Adjusted update bat script to clear dropins, lib, and pull patch from updated location. Added installer build instructions to README files.

rma-tom reviewed (dismissed):
Reviewed and approved changes for this update

jmdegeorge committed:
Corrected release date and simpleconfigurator bundle.info directory.

jmdegeorge commented:
@hxiedwr , I have updated the patch build system and .bat file to use produce the patch and pull it from the updated file location. 

Currently, the release date file and patch are hard coded to match file names in the links you provided. 

- releasedate_v2.2.0_basis.log
- wrims_patch_v2.2.0_basis.zip

https://github.com/CentralValleyModeling/wrims/releases/tag/20250103

Does this match what you'd expect from the build system? Or should the patch and release file names include the version number from the release like this:

- releasedate_<version>_basis.log
- wrims_patch_<version>_basis.zip

hxiedwr commented:
Can you please update the attached file to both package and patch file. I renamed the file to a text file and you may have to rename it back.
[setting.prf.txt](https://github.com/user-attachments/files/18322716/setting.prf.txt)

hxiedwr reviewed (dismissed):
I am not sure if everything is ok but we can test them later.

jmdegeorge committed:
Adding setting.prf file with updated values per Hao's request.

jmdegeorge commented:
> Can you please update the attached file to both package and patch file. I renamed the file to a text file and you may have to rename it back. [setting.prf.txt](https://github.com/user-attachments/files/18322716/setting.prf.txt)

@hxiedwr  I found the setting.prf file in the /data folder of the installer and added it to the new build configuration with your updated settings.

jmdegeorge commented on wrims-gui/src/main/resources/installer/data/setting.prf:
@hxiedwr , can you confirm this is the correct file you wanted and then I'll push the changes and make a new release.

hxiedwr replied:
Yes. The number need to be changed to 1000000.

hxiedwr replied:
Yes. The number need to be changed to 1000000.

hxiedwr reviewed (commented):
approved

hxiedwr reviewed (commented):
approved.

hxiedwr reviewed (approved):
approved
